lib/Elastica/Aggregation/ExtendedStatsBucket.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Elastica\Aggregation;
+
+use Elastica\Exception\InvalidException;
+
+/**
+ * @see https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-pipeline-extended-stats-bucket-aggregation.html
```

Comment:
Can you add just a title? 

`Implements a Extended Stats Bucket Aggregation` would be enough :)